frontend/components/Course/InstructorQueuePage/Queue.tsx

```diff
@@ -22,6 +32,7 @@ const Queue = (props: QueueProps) => {
     const {
         courseId,
         queue,
+        activeStaff,
```

Comment:
I don't think we want activeStaff to be queried just on page load - you should write a custom hook (in `hooks/data-fetching`) that calls useResource with this value as the initial value and the activeStaff route as the URL, and set up a refresh interval so we make the API call every 30 seconds. This pattern is pretty common throughout the codebase - let me know if you have any questions about this.

You'll have to do the same thing for StudentQueue, although you should be able to share the hook.